weave/tests/test_arrow.py

```diff
@@ -1654,218 +1655,17 @@ def test_save_nested_custom_objs():
-    for type_ndx, type_example in enumerate(all_types):
-        res_with_data.append((f"{str(type_ndx).zfill(3)}_empty", type_example, []))
-        if type_example.assign_type(types.NoneType()) and not contains_unknown(
-            type_example
-        ):
-            res_with_data.append(
-                (f"{str(type_ndx).zfill(3)}_null", type_example, [None])
-            )
-
-    res_with_concat: list[tuple[types.Type, list, types.Type, list]] = []
-    names: list[str] = []
-    for data_example in res_with_data:
-        names.append(f"{str(len(names)).zfill(3)}-t_{data_example[0]}_concat_none")
-        res_with_concat.append(
-            (data_example[1], data_example[2], types.NoneType(), [None])
-        )
-        if not contains_unknown(data_example[1]):
-            names.append(
-                f"{str(len(names)).zfill(3)}-t_{data_example[0]}_concat_jumbled"
-            )
-            res_with_concat.append(
-                (data_example[1], data_example[2], jumble_type(data_example[1]), [None])
-            )
-
-    return names, res_with_concat
-
-
-names, cases = make_identity_permutations(all_types)
-
-# This test hits the list<dense union> issue described in `safe_replace_with_mask`.
-# We don;t believe this is possible to hit in practice today, so skipping for now.
-skip_indicies = [280, 281]
-# skip_indicies = []
-
-cases = [c for ndx, c in enumerate(cases) if ndx not in skip_indicies]
-names = [n for ndx, n in enumerate(names) if ndx not in skip_indicies]
-
-
-@pytest.mark.parametrize(
-    "assumed_weave_type, data, concat_with_weave_type, concat_with_data",
-    cases,
-    ids=names,
-)
-def test_identity_awl_operations_3(
-    assumed_weave_type, data, concat_with_weave_type, concat_with_data
-):
-    # ArrowWeaveList doesn't allow mergeable unions in its
-    # ObjectType anymore, so we need to merge them here.
-    if isinstance(assumed_weave_type, types.UnionType):
-        assumed_weave_type = types.merge_many_types(assumed_weave_type.members)
-    if isinstance(concat_with_weave_type, types.UnionType):
-        concat_with_weave_type = types.merge_many_types(concat_with_weave_type.members)
-
-    print("ASSUMED WEAVE TYPE", assumed_weave_type)
-    print("DATA", data)
-    print("CONCAT WITH WEAVE TYPE", concat_with_weave_type)
-    print("CONCAT WITH DATA", concat_with_data)
-    # Assert basic in-memory identity
-    raw_awl = arrow.to_arrow(data, types.List(assumed_weave_type))
-    assert raw_awl.to_pylist_raw() == data
-
-    # Assert save/load identity
-    awl_node = weave.save(raw_awl)
-    assert weave.use(awl_node).to_pylist_raw() == data
-
-    # Assert concat-ability
-    raw_concat_awl = arrow.to_arrow(
-        concat_with_data, types.List(concat_with_weave_type)
-    )
-    concat_awl = weave.save(raw_concat_awl)
-    concatted = ops.make_list(a=awl_node, b=concat_awl).concat()
-    assert weave.use(concatted).to_pylist_raw() == data + concat_with_data
-
```

Comment:
why are we removing this?

Reply:
This tests generates a bunch of invalid states that we can't really get to anymore, so I removed it.